Dockerimage/Dockerfile

```diff
@@ -0,0 +1,81 @@
+# Elastalert Docker image running on Alpine Linux.
+# Build image with: docker build -t ivankrizsan/elastalert:latest .
```

Comment:
@krizsan you referenced the old repo here.

Reply:
Sorry, missed that one. Fixed.